packages/apps/desktop-viewer-test/src/backend/main.ts

```diff
@@ -42,6 +43,7 @@ const viewerMain = async () => {
       scope,
       redirectUri:
         redirectUri && redirectUri.length > 0 ? redirectUri : undefined,
+      issuerUrl,
```

Comment:
what happens if this is empty/undefined?....I believe it will end up as an empty string as is and may not be handled as falsey in imjs, resulting in an error.  This was the case for redirectUri (overriding the default with empty string), thus the additional condition. I would check what happens in this case and maybe account for it as needed

Reply:
Great catch! Fixed